assets/javascripts/discourse/components/ai-bot-header-panel.js

```diff
@@ -0,0 +1,45 @@
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import Component from "@ember/component";
```

Comment:
would be nice to just use "@glimmer/component" here

Reply:
done :)